recipes/cath-tools/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "cath-tools" %}
+{% set version = "0.16.5" %}
+{% set sha256 = "10b2913edf70cda698a47003d4f1ad8de89fc73ccacdafb1d75ce444be25bffe" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://github.com/UCLOrengoGroup/cath-tools/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake >3.2
+    - gsl >=2.6
+    - llvm-openmp  # [osx]
+    - perl >=5.08
+    - pkgconfig
+    - pthread-stubs
+    - boost-cpp
+
+
+test:
+  source_files:
+    - build-test-data
+    - perl
+  commands:
+    - build-test
```

Comment:
Hm, now I realize that the `build-test` binary should probably not actually be part of the package. Is it only for testing whether the compilation was succesful? If you want to run unit tests, you can do this at the end of the build script. Most Bioconda recipes usually don’t run a program’s full set of unit tests, so you could also skip them. It is somewhat nicer to have them, but since they could take a long time to run and it is sometimes difficult to run them, generally the tests in recipes are allowed to be more basic.

Reply:
Ok I will move it at the end of the build script

Reply:
(And please remove it from `CATH_PROGRAMS`)

Reply:
Yes sure